content/data.md

```diff
@@ -19,6 +21,53 @@ giving a rough approximation that assumes binary gender categories.
 
 ## Packages
 
+### Maintainer Gender 2017
+
+In [Pinto, Wiese & Dias (2018). How Do Scientists Develop Scientific Software? An External Replication](http://gustavopinto.org/lost+found/saner-rene2018.pdf), the authors surveyed CRAN maintainers that self-identified as a 
+scientist. The survey was conducted starting in late November 2017 and ending in early January 2018. The respondents were contacted 
+by email and asked to complete an online questionnaire, with a response rate of 24% (a total of 1,553 responses). 
+
+<div class="commentbox"> In 2017, 10.9% of package maintainers who identified as scientists were women. </div>
+
+- 44.6% of respondents identify between 30 and 40 years of age at the time of the survey.
+
+- Overwhelmingly, 80.0% of respondents identified as either having completed or are in progress of completing a doctorate degree. 
+
+- Almost half (48.9%) of the respondents identify as being located in Europe and another 34.1% as being located in North America.
+
+<br>
+
+<div style="display: flex; justify-content: center">
+<img src="../images/pinto_et_al_2018_gender_plot.png" 
+    alt="In 2017, of the package author surveyed, 10.9% of respondents self-identify as female, about 88.2% identify as male, 0.1% of respondents self-identify as non-binary."
+    width = "425"/>
+<img src="../images/pinto_et_al_2018_age_cats_plot.png" 
+    alt="44.6% of respondents identify between 30 and 40 years of age at the 
+    time of the survey. 23.1% identify being younger than 30 and 31.7% as being 
+    older than 40 years of age. The remainder did not provide an answer. "
+    width = "425"/>
+</div>   
+
+<br>
+
+<div style="display: flex; justify-content: center">
+<img src="../images/pinto_et_al_2018_educ_plot.png" 
+    alt="80.0% of respondents identify as either having completed or are in 
```

Comment:
having completed or are in progress of completing a doctorate degree -> having a completed or in-progress doctorate degree.

Reply:
Incorporated in 56e8927